lang/lang.go

```diff
@@ -159,6 +161,15 @@ func AddTranslationsFS(fs embed.FS, dir string) (retErr error) {
 	return retErr
 }
 
+// SetPreferredLanguage allows an app to set the preferred language for translations, overwriting the System Locale.
```

Comment:
Since this adds a new public API, it will need to be targeted for Fyne 2.6 and should have a `Since: 2.6` line in the comment (see other APIs for an example). Also the PR should be based against develop instead of the release branch

Reply:
This builds on top of this bugfix https://github.com/fyne-io/fyne/pull/5016
which is i think not yet merged back into develop.

Once that is done, i am happy to base it against develop.
Or should i merge #5016 into develop myself?

Reply:
I will leave it to @andydotxyz to manage the branches since I'm not sure what our usual process is for keeping `develop` in sync with `release/v2.x.x` branches

Reply:
Release branch goes back into develop after each point release, so it should have been done a few weeks back.